### PR TITLE
K8SPXC-1363 Update for PXC operator release 1.15.0

### PR DESCRIPTION
**Pull request by ptankov**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by jvpasinatto — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by eleo007 — DISMISSED** [review]

File: api-tests/operator_route_test.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func TestOperatorRouteShouldReturnRightOperatorVersion(t *testing.T) {
 		{"pxc-operator", "1.11.0"},
 		{"pxc-operator", "1.12.0"},
 		{"pxc-operator", "1.13.0"},
+		{"pxc-operator", "1.15.0"},
```
Comment (tplavcic) [inline api-tests/operator_route_test.go:29]: maybe put it after 1.14.0

File: sources/operator.1.15.0.pxc-operator.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          },
+          "5.7.34-31.51": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.34-31.51",
+            "image_hash": "f8d51d7932b9bb1a5a896c7ae440256230eb69b55798ff37397aabfd58b80ccb",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.42.0": {
```
Comment (tplavcic) [inline sources/operator.1.15.0.pxc-operator.json:76]: do we want to have 3 versions of PMM here, usually we just kept one?

File: sources/operator.1.15.0.pxc-operator.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+          "2.8.5": {
+            "image_path": "percona/haproxy:2.8.5",
+            "image_hash": "fe63b0e7d503b25ab3d6f4aeb2e983868fe50bf85ebc91c713a990306718a8a2",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "backup": {
+          "8.0.35": {
+            "image_path": "perconalab/percona-xtradb-cluster-operator:1.15.0-pxc8.0-backup-pxb8.0.35",
```
Comment (tplavcic) [inline sources/operator.1.15.0.pxc-operator.json:113]: should we put full PXC version in image as well?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: sources/operator.1.15.0.pxc-operator.json (hunk 3 above)
Comment (hors) [reply sources/operator.1.15.0.pxc-operator.json:113]: we did not do it earlier and I do not want to have it only for one release (we have a plan to use official image starting from next release)

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.15.0.pxc-operator.json [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        "pmm": {
+          "2.42.0": {
+            "image_path": "percona/pmm-client:2.42.0",
+            "image_hash": "14cb96de47e3bc239bf285f22ec6f170b4a1181301b19100f5b7dc22c210bf8c",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "proxysql": {
+          "2.6.3": {
```
Comment (tplavcic) [inline sources/operator.1.15.0.pxc-operator.json:84]: From what I see in PXC 8.0.36 distribution Proxysql 2.5.5 is available, not sure how we got this 2.6.3 version.

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: sources/operator.1.15.0.pxc-operator.json (hunk 4 above)
Comment (hors) [reply sources/operator.1.15.0.pxc-operator.json:84]: In this case, we can be 100% sure it was tested with PXC 8.0.36

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by hors — APPROVED** [review]

**Review by eleo007 — APPROVED** [review]

**Review by eleo007 — DISMISSED** [review]

**Review by eleo007 — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Pull request merged**